3-extra/2-array-of-objects.js

```diff
@@ -9,74 +9,104 @@
     Implement a function which takes the array of books as a parameter, and returns an array of book titles.
     Each title in the resulting array should be the highest rated book in its genre.
 */
+let highestBooks = [];
+let highBook = {};
+let finalList = [];
```

Comment:
Issue: While you can declare variables like this in the body of a script it's generally bad practice for this type of script. (We call these globals) Instead you should try and declare your variables in the function body and if you need to share variables between functions pass them around in parameters.